Probability: Class notes (#14)
infobayes/ScientificComputingNewApproach: chaps/probability-class-notes.tex

% The development in question aims to present the main fundamentals of probability. The student Gabriel Aguiar did not seek to fit the text into the predefined body of the "Probability" chapter.

\documentclass{article}
\usepackage[utf8]{inputenc}

\title{Probability fundamentals}
\author{Gabriel Aguiar}
\date{August 2019}

\begin{document}

\maketitle

\section{Formulations}

\begin{itemize}
    

> \item We define like Sample space $\Omega$ a set of elementary events.

Reviewer: ```suggestion
\item We define as Sample space $\Omega$ a set of elementary events.
```

> \item We define like Sigma-algebra $\Sigma$ a set of $\Omega$ subsets. $\Sigma$ shows the following properties: set of random events; $\Omega$ belongs to $\Sigma$; if A and B belong to $\Sigma$, their union and their intersection also belong.

Reviewer: ```suggestion
\item We define as Sigma-algebra $\Sigma$ a set of $\Omega$ subsets. $\Sigma$ shows the following properties:
\begin{enumerate}
    \item set of random events;
    \item $\Omega$ belongs to $\Sigma$;
    \item if A and B belong to $\Sigma$, their union and their intersection also belong.
\end{enumerate}
```

> \item We define like probability a function P: $\Sigma$ $\rightarrow$ [0, 1]. P presents the following properties: P($\Omega$) = 1; if A and B are $\Sigma$ disjoint subsets, P(A $\cup$ B) = P(A) + P(B).

Reviewer: ```suggestion
\item We define as probability a function P: $\Sigma$ $\rightarrow$ [0, 1]. P presents the following properties: 
\begin{enumerate}
\item P($\Omega$) = 1; 
\item if A and B are $\Sigma$ disjoint subsets, P(A $\cup$ B) = P(A) + P(B).
\end{enumerate}
```

> \item P(B/A) = $\frac{1}{P(A)}$ P(A $\cap$ B).

Reviewer: ```suggestion
\item P(B/A) $\doteq \frac{1}{P(A)}$ P(A $\cap$ B).
```

Author: Can you explain more about this suggestion?
